lib/quest/rspec_runner.rb

```diff
@@ -0,0 +1,35 @@
+module Quest
+  class RSpecRunner
+    # Based loosely on https://github.com/guard/guard-rspec/blob/master/lib/guard/rspec/rspec_process.rb
+
+    attr_reader :result
+
+    def initialize(spec_file, spec_helper, tmp_status_file)
+      @spec_file       = spec_file
+      @spec_helper     = spec_helper
+      @tmp_status_file = tmp_status_file
```

Comment:
Yeah, that makes sense. I was setting these things up to allow the tempfile to be potentially configurable, but using a ruby Tempfile will make that irrelevant.